iOS build failing because of connectivity_plus

### What
Analyzing dependencies
Adding spec repo `trunk` with CDN `[https://cdn.cocoapods.org/`](https://cdn.cocoapods.org/%60)
[!] CocoaPods could not find compatible versions for pod "connectivity_plus":
  In Podfile:
    connectivity_plus (from `.symlinks/plugins/connectivity_plus/ios`)

Specs satisfying the `connectivity_plus (from `.symlinks/plugins/connectivity_plus/ios`)` dependency were found, but they required a higher minimum deployment target.
Error: Process completed with exit code 1.

--- a/packages/smooth_app/ios/Podfile
+++ b/packages/smooth_app/ios/Podfile
@@ -1,5 +1,5 @@
 # Uncomment this line to define a global platform for your project
-platform :ios, '11.0'
+platform :ios, '12.0'
 
 # CocoaPods analytics sends network stats synchronously affecting flutter build latency.
 ENV['COCOAPODS_DISABLE_STATS'] = 'true'

